modules/uhttp/client_filters.go

```diff
@@ -0,0 +1,81 @@
+package uhttp
+
+import (
+	"net/http"
+
+	"github.com/opentracing/opentracing-go"
+
```

Comment:
nit: no newline here. put line 9 up here and separate it with a space from opentracing